SU-HSDO/suhumsci
# SHS-4881: Traditional Vertical Linked Card body background color incorrect

# READY FOR REVIEW

## Summary
- Removes BG color for vertical linked cards and ensures the card title or caption are stretched when utility class is present

## Need Review By (Date)
4/11

## Urgency
medium

## Steps to Test
1. Go to https://hs-traditional.suhumsci.loc/components/collections-cards-uniform-and-stretch-1 and verify the title/caption is stretched so the cards are all uniform
2. Edit the page and randomly edit various cards to include long title, short titles, long body copy, and short body copy. Verify the Vertical linked cards are all extending and all cards are equal height within a given collection
![Screenshot from 2023-04-10 09-24-13](https://user-images.githubusercontent.com/369018/230920580-431b761c-f269-486b-8583-2bd0ae999c5e.png)


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
